src/compressed_tensors/compressors/sparse_compressors/base.py

```diff
@@ -97,8 +107,10 @@ def decompress(
         :param device: device to load decompressed weights onto
         :return: iterator for generating decompressed weights
         """
-        weight_mappings = get_nested_weight_mappings(
-            path_to_model_or_tensors, self.COMPRESSION_PARAM_NAMES
+        weight_mappings, other_params = get_nested_weight_mappings(
+            path_to_model_or_tensors,
+            self.COMPRESSION_PARAM_NAMES,
```

Comment:
All compressor define this except Dense, Ideally this would be a property that must be implemented and enforced via BaseCompressor. This will need a change for all compressors, I will make that in a follow up PR to not blow up the scope

Reply:
This is for self. COMPRESSION_PARAM_NAMES?